api/src/test/java/io/opentelemetry/correlationcontext/propagation/CorrelationsContextUtilsTest.java

```diff
@@ -24,40 +24,37 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/** Unit tests for {@link ContextUtils}. */
+/** Unit tests for {@link CorrelationsContextUtils}. */
 @RunWith(JUnit4.class)
-public final class ContextUtilsTest {
+public final class CorrelationsContextUtilsTest {
   @Test
-  public void testGetCurrentCorrelationContex_DefaultContext() {
-    CorrelationContext distContext = ContextUtils.getValue(Context.current());
-    assertThat(distContext).isNotNull();
-    assertThat(distContext.getEntries()).isEmpty();
+  public void testGetCurrentDistributedContext_DefaultContext() {
```

Comment:
Why `DistributedContext` in the name?

Reply:
Oops, one of the remnants of the old code (that I didn't detect)